modules/oe_theme_content_publication/templates/oe-theme-content-publication-collections.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+/**
+ * @file
+ * Theme template for Publication collection extra field.
+ *
+ * @see \Drupal\oe_theme_content_publication\Plugin\ExtraField\Display\PublicationCollection::viewElements()
+ */
+#}
+{% for item in items %}{{ item }}{% if not loop.last %} | {% endif %}{% endfor %}
```

Comment:
```suggestion
{% include '@oe_theme/field/field--multiple-values--pipe-separated.html.twig' %}
```
to have single template for all fields separated by pipe

Reply:
Let's keep here this as custom template instead of forcing the field template on a custom extra field.